Cargo.toml

```diff
@@ -49,6 +49,7 @@ tracing = "0.1.40"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 uts2ts = "0.4.1"
 path-clean = "1.0.1"
+chrono = "0.4.38"
```

Comment:
Is this dependency being used?

Reply:
It was not used in the end. Forgot to remove it, sorry.

Reply:
Sorry, I was busy with somwthing else. No, `chrono` has not been and is not used. It was one of the reasons why I created the uts2ts crate. ;-)